### ISSUE TITLE
Permit fx/sub'ing to sequence of keys

### ISSUE THREAD

**Issue by fdeitylink** [cljfx/cljfx#84]
As it stands, you can only `fx/sub` to a top-level key. I think it would be useful if we could supply a sequence of keys to `fx/sub` to subscribe to arbitrarily nested values. Something like `(fx/sub context :a :b :c)`. This would enable smarter memoization and reduce recomputation even more than is already provided. Instead of having recompute when the top-level key changes, it would only be needed when the full key sequence changes. The syntax would also be more convenient than having to write `(get-in (fx/sub context :a) [:b :c ...])`

~~As far as implementing it goes, I think we would need to change `cljfx.context/sub` to pass the entirety of `k & args` to `key-sub` and change `key-sub` to use `get-in` instead of `get`.~~

EDIT: Just saw issue #69 for subscribing to nested elements, where you recommend granular subscription functions. Made this.
```clojure
(defn- keys-sub
  [ctx & ks]
  (let [k (last ks)
        ;; deps is all the "parent" keys
        deps (butlast ks)]
    (get (apply sub ctx deps) k)))
```
And then in `sub`
```clojure
(let [sub-id (if (fn? k)
               (apply vector k args)
               (apply vector keys-sub k args))
      ...]
  ...)
```

We can rename `deps` if you feel that's appropriate. I know the `::*deps` key is used in the context internals, so maybe you'd want a different name to avoid confusion.

I can make a PR if that sounds good to you.

**Comment by vlaaad** [comment]
Ohh, I'm always nervous about touching context stuff, all those caches and invalidations are very brittle and hard to think about.
With that said, I also thought that specifying multiple keys, as well as specifying 0 keys, makes sense. I don't remember why I did context that way, to be honest. Maybe it had something to do with how the initial implementation of cache invalidation worked. I think this restriction in context can be lifted, there is already a [key-sub](https://github.com/cljfx/cljfx/blob/master/src/cljfx/context.clj#L106) in `context.clj` that can be extended to use get-in and receive remaining args. PR welcome!

**Comment by vlaaad** [comment]
By the way, we should make `key-sub` use `get-in` instead of recursive subscriptions: the overhead of associated caching is higher than the price of executing a `get`.

**Comment by vlaaad** [comment]
Released 1.7.5 that addresses your issue — see [changelog](https://github.com/cljfx/cljfx/blob/master/CHANGELOG.md#175---2020-08-12) for details.
